### PR TITLE
Add StatefulSet example

### PR DESCRIPTION
**Pull request by hausdorff**

### CODE REVIEW

File: kubernetes-ts-helm-wordpress/README.md [hunk 1]
```diff
@@ -46,7 +46,17 @@ $ pulumi stack init
...
+This chart requires you to set two configuration values: `wordpressPassword`, `mariadbPassword`, and
+`mariadbRootPassword`. If these values are not set, Helm will re-generate the passwords each
+upgrade, which will cause the upgrade to fail. (See
+[helm/charts#5167](https://github.com/helm/charts/issues/5167).)
+
+```sh
+p config set isMinikube false # This chart disallows minikube.
+p config set wordpressPassword <value>
+p config set mariadbPassword <value>
+p config set mariadbRootPassword <value>
```
Comment (lukehoban) [inline kubernetes-ts-helm-wordpress/README.md]: This isn't a standard alias - we should use `pulumi config` here.

File: kubernetes-ts-stateful/cassandra.ts [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    transforms?: [(a: inputApi.apps.v1.StatefulSet) => void];
+}
+
+export class Cluster extends k8s.apps.v1.StatefulSet {
+    constructor(name: string, args?: ClusterOpts, opts?: pulumi.CustomResourceOptions) {
+        super(name, base(args), opts);
+    }
+}
+
+function base(opts?: ClusterOpts): inputApi.apps.v1.StatefulSet {
```
Comment (lukehoban) [inline kubernetes-ts-stateful/cassandra.ts]: Could `base` have a nicer name?

File: kubernetes-ts-stateful/cassandra.ts [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    resourceRequest?: pulumi.Input<object>;
+    securityCapabilities?: pulumi.Input<inputApi.core.v1.Capabilities>;
+    heapSize?: pulumi.Input<string>;
+    heapNewSize?: pulumi.Input<string>;
+    volumeClaimRequests?: pulumi.Input<object>;
+
+    transforms?: [(a: inputApi.apps.v1.StatefulSet) => void];
+}
+
+export class Cluster extends k8s.apps.v1.StatefulSet {
```
Comment (lukehoban) [inline kubernetes-ts-stateful/cassandra.ts]: Perhaps a comment on this?

File: kubernetes-ts-stateful/package.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "k8s-nginx",
+  "version": "0.1.0",
+  "devDependencies": {
+    "@types/node": "^9.3.0"
+  },
+  "dependencies": {
+    "@pulumi/kubernetes": "^0.16.0",
+    "@pulumi/pulumi": "^0.15.0"
```
Comment (lukehoban) [inline kubernetes-ts-stateful/package.json]: Move this to `0.16.0`, and maybe Kubernetes to `0.17.0`?

Comment (ellismg) [reply kubernetes-ts-stateful/package.json]: We probably just want to be using `latest` here, as with our other examples right?  That way we don't have to keep updating these examples every time we bump (minor) package versions?

**Comment by mikhailshilkov** [comment]
@hausdorff Do you remember why this was never finished? Worth adding last bits and merge?

**Comment by justinvp** [comment]
@hausdorff, what's the status of this PR?

**Comment by hausdorff** [comment]
This PR is effectively abandoned. We originally started this when people were asking for `StatefulSet` examples, but I don't think it's particularly well-written, and I don't think it elucidates anything that's particularly mysterious.

If there is demand for it I can fix it up but this is not something that I think is particularly high leverage. LMK what you think.

**Comment by justinvp** [comment]
> This PR is effectively abandoned.

OK, let's close the PR then.

**Comment by stack72** [comment]
Closed as abandoned - thanks for clearing this up @hausdorff

**Pull request closed without merging**